raiden_contracts/contract_manager.py

```diff
@@ -76,8 +76,10 @@ def _compile_all_contracts(self) -> None:
 
         def relativise(path):
             return path.relative_to(_BASE)
-        import_dir_map = ['=.'] + ['%s=%s' % (k, relativise(v))
-                                   for k, v in self.contracts_source_dirs.items()]
+        import_dir_map = [
+            '%s=%s' % (k, relativise(v))
+            for k, v in self.contracts_source_dirs.items()
+        ]
```

Comment:
Explanation:
From https://solidity.readthedocs.io/en/v0.5.2/using-the-compiler.html?highlight=remappings#using-the-commandline-compiler:

> An empty remapping prefix is not allowed.

Not sure why `['=.']` was added in the first place, it is enough to compute `['raiden=contracts', 'test=contracts/test', 'services=contracts/services']`